styles.css

```diff
@@ -0,0 +1,244 @@
+/* Template 1- Mobile First Styles */
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* body */
+body {
+  position: relative;
+  min-width: 375px;
+  background: #f7f7f9;
+  height: 100vh;
+  margin: 0;
+}
+
+/* Header */
+.header {
+  background-color: #fff;
+  width: 100%;
+  height: 92px;
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  padding: 10px;
+  box-sizing: border-box;
+}
+
+header {
+  background-image: none;
+}
+
+nav {
+  display: none;
+}
+
+/* My logo */
+.my-logo {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+  height: 20px;
+}
+
+/* Headline */
+.headline {
+  position: absolute;
+  width: 100%;
+  height: 604px;
+  top: 92px;
+  background-image: url(./images/header-shapes-mobile.png);
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 20px;
+  box-sizing: border-box;
+  padding: 112px 24px;
+}
```

Comment:
- [ ] Kindly, add a `border-bottom-left-radius` to  you headline to match the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=yDKg5dMjRcmmZ2y5-0). This will make you web site look way more better (as the figma design is done by professional web designer) 😃
**You project headline**
![image](https://user-images.githubusercontent.com/74411135/227725248-a3d0d142-a338-475b-ae5b-3b69e3b12684.png)

**Figma design headline**
![image](https://user-images.githubusercontent.com/74411135/227725276-7c1acbea-effd-49e8-b1e3-716e17a91082.png)

Reply:
I couldn't find the dimension earlier but i have fixed it now.